### PR TITLE
migrated form response data to react

### PR DESCRIPTION
**Pull request by stephanlamoureux**

### CODE REVIEW

File: job-search-react/src/components/Header/Header.js [hunk 1]
```diff
@@ -119,7 +119,7 @@ export default function Header(props) {
 									<i
 										alt="moon icon"
 										className={`fas fa-2x fa-${
-											props.theme === 'light' ? 'sun' : 'moon'
+											props.theme !== 'light' ? 'sun' : 'moon'
```
Comment (axecopfire) [inline job-search-react/src/components/Header/Header.js:122]: Why is this in here?

File: job-search-react/src/components/JobApp/Form/Distance/Distance.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-export default function Distance() {
```
Comment (axecopfire) [inline job-search-react/src/components/JobApp/Form/Distance/Distance.js:3]: Are we using this new props variable?

File: job-search-react/src/components/JobApp/Form/Form.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ import Distance from './Distance/Distance';
 export default function Form(props) {
 	return (
 		<form
-			onSubmit={(event) => props.data(event)}
```
Comment (axecopfire) [inline job-search-react/src/components/JobApp/Form/Form.js:10]: Do we need these extra curly brackets?

File: job-search-react/src/components/JobApp/Form/Remote/Remote.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-export default function Checkbox() {
```
Comment (axecopfire) [inline job-search-react/src/components/JobApp/Form/Remote/Remote.js:3]: Do we need props?

File: job-search-react/src/components/JobApp/Form/ZipCode/ZipCode.js [hunk 5]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-export default function ZipCode() {
```
Comment (axecopfire) [inline job-search-react/src/components/JobApp/Form/ZipCode/ZipCode.js:3]: refer to props comments

File: job-search-react/src/components/JobApp/JobApp.js [hunk 6]
```diff
@@ -3,12 +3,22 @@ import Typed from 'react-typed';
...
 	function formData(clickEvent) {
 		clickEvent.preventDefault();
-		setUserData(clickEvent);
-		console.log(userData);
+		const formResponse = {
+			zipCode: clickEvent.target[0].value,
+			remote: clickEvent.target[1].checked,
+			distance: clickEvent.target[2].value
+		};
+		console.log(clickEvent.target[0].value);
```
Comment (axecopfire) [inline job-search-react/src/components/JobApp/JobApp.js:19]: Can we get rid of this? log

File: job-search-react/src/components/Header/Header.js (hunk 1 above)
Comment (stephanlamoureux) [reply job-search-react/src/components/Header/Header.js:122]: because that's how its supposed to be

**Review by axecopfire — APPROVED** [review]

**Pull request merged**